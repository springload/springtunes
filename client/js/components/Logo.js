import React from 'react';

const Logo = () => (
    <div className="logo">
        <img
            src="../img/springtunes.svg"
            alt="Springtunes Logo"
            width="250"
            height="41"
        />
    </div>
);

export default Logo;
