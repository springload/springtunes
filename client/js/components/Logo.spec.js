import React from 'react';
import Logo from './Logo';
import renderer from 'react-test-renderer';

describe('Logo component', () => {
    it('renders correctly with min params', () => {
        const result = renderer.create(<Logo />).toJSON();
        expect(result).toMatchSnapshot();
    });
});
