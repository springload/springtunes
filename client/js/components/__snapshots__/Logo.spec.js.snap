exports[`Logo component renders correctly with min params 1`] = `
<div
  className="logo">
  <img
    alt="Springtunes Logo"
    height="41"
    src="../img/springtunes.svg"
    width="250" />
</div>
`;
